docs/content/guide/component-router.ngdoc

```diff
@@ -5,6 +5,11 @@
 
 # Component Router
 
+<div class="alert alert-danger">
+**Deprecation Notice:** The Component Router (ngComponentRouter module) has been deprecated and will not receive further updates.
```

Comment:
Perhaps we should be saying... 

> This implementation of the component router has been deprecated ... 

So that it doesn't make it sound like there will never be a component router